engineering_tools/carma_script_extensions/cloud-sim

```diff
@@ -126,7 +130,7 @@ cloud-sim__start() {
         if [ -f "$CARMA_CLOUD_SIM_CLOUD_FORMATION_PATH" ]; then rm "${CARMA_CLOUD_SIM_CLOUD_FORMATION_PATH}"; fi
 
         # Pull new file
-        wget -O "${CARMA_CLOUD_SIM_CLOUD_FORMATION_PATH}" "https://raw.githubusercontent.com/usdot-fhwa-stol/carma-platform/feature/aws-xil/engineering_tools/xil-cloud-formation-v0.yaml"
+        wget -O "${CARMA_CLOUD_SIM_CLOUD_FORMATION_PATH}" "https://raw.githubusercontent.com/usdot-fhwa-stol/carma-platform/feature/cloud-sim/engineering_tools/xil-cloud-formation-v0.yaml"
```

Comment:
Will this URL break when we merge/delete this branch?

Reply:
Resolved